### DISCUSSION TITLE
Unable to apply Our School In Github Campus Education Program

### DISCUSSION THREAD

**Discussion by lightoflifetrust** [community/community#127091] [GitHub Education]
### Hi everyone,

Dear Github Education Support Team
When we are trying to apply for our school in GitHub education program, we are getting error.
Error:  **Partner school application order form has an invalid content type**

![Screenshot_6](https://github.com/community/community/assets/163536268/39fb2cc4-86eb-44b1-ad28-10f83d6cc16f)

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation, you need to verify it!!


Access the [Github Support](https://support.github.com/)

Click on My Ticket -> New Ticket -> Finally describe it

If I supported you consider to mark my answer. Good Luck!

**Reply by lightoflifetrust** [reply]
We are already raised ticket to GitHub support team.
But there is no one to respond on our query.

**Comment by davevad93** [comment]
Hi @lightoflifetrust ,

Is the problem occurring when trying to upload the school's logo? If that's the case, it could be due to the file format, the error `Invalid content type` suggests that the logo you are uploading is neither a `JPEG` nor a `PNG` file. If that's not the problem or the file extension is correct and the error persists, you should contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain thoroughly your issue.

**Comment by lightoflifetrust** [comment]
Its not issue of logo and .
We are already raised ticket to GitHub support team.
But there is no one to respond on our query.

**Reply by davevad93** [reply]
That's unfortunate.. I hope that support will answer you as soon as possible.
